.github/workflows/build.yml

```diff
@@ -0,0 +1,137 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'bld/bin/**'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - 'bld/bin/**'
```

Comment:
The pipeline is triggered for pull requests targeting `master` branch and pushes to `master` branch. It intentionally ignores commits that touch only `bld/bin` directory to prevent loops if the binaries were automatically updated as part of the pipeline (not done at the moment).